### PR TITLE
update description of cronjob settings to be aligned to the documenta…

### PR DESCRIPTION
**Pull request by szaimen**
…tion update

See https://github.com/nextcloud/documentation/pull/8209

Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

File: apps/settings/templates/settings/admin/server.php [hunk 1]
```diff
@@ -71,7 +71,7 @@
 	   title="<?php p($l->t('Open documentation'));?>"
 	   href="<?php p(link_to_docs('admin-background-jobs')); ?>"></a>
 
-	<p class="settings-hint"><?php p($l->t('For optimal performance it\'s important to configure background jobs correctly. For bigger instances \'Cron\' is the recommended setting. Please see the documentation for more information.'));?></p>
+	<p class="settings-hint"><?php p($l->t('For Nextcloud to work correctly, it\'s important to configure background jobs correctly. \'Cron\' is the recommended setting. Please see the documentation for more information.'));?></p>
```
Comment (nickvergessen) [inline apps/settings/templates/settings/admin/server.php:74]: Please avoid labelling :)

```suggestion
	<p class="settings-hint"><?php p($l->t('For the server to work correctly, it\'s important to configure background jobs correctly. \'Cron\' is the recommended setting. Please see the documentation for more information.'));?></p>
```

Comment (szaimen) [reply apps/settings/templates/settings/admin/server.php:74]: fine by me :)

Comment (szaimen) [reply apps/settings/templates/settings/admin/server.php]: done

File: apps/settings/templates/settings/admin/server.php [hunk 2]
```diff
@@ -71,7 +71,7 @@
 	   title="<?php p($l->t('Open documentation'));?>"
 	   href="<?php p(link_to_docs('admin-background-jobs')); ?>"></a>
 
-	<p class="settings-hint"><?php p($l->t('For optimal performance it\'s important to configure background jobs correctly. For bigger instances \'Cron\' is the recommended setting. Please see the documentation for more information.'));?></p>
+	<p class="settings-hint"><?php p($l->t('For the server to work correctly, it\'s important to configure background jobs correctly. \'Cron\' is the recommended setting. Please see the documentation for more information.'));?></p>
```
Comment (Valdnet) [inline apps/settings/templates/settings/admin/server.php:74]:
```suggestion
	<p class="settings-hint"><?php p($l->t('For the server to work properly, it\'s important to configure background jobs correctly. \'Cron\' is the recommended setting. Please see the documentation for more information.'));?></p>
```

File: apps/settings/templates/settings/admin/server.php [hunk 3]
```diff
@@ -81,15 +81,15 @@
 			print_unescaped('checked="checked"');
 		} ?>>
 				<label for="backgroundjobs_ajax">AJAX</label><br/>
-				<em><?php p($l->t("Execute one task with each page loaded.")); ?></em>
+				<em><?php p($l->t("Execute one task with each page loaded. Usecase: Single user instance.")); ?></em>
```
Comment (Valdnet) [inline apps/settings/templates/settings/admin/server.php:84]:
```suggestion
				<em><?php p($l->t("Execute one task with each page loaded. Use case: Single user instance.")); ?></em>
```

File: apps/settings/templates/settings/admin/server.php [hunk 4]
```diff
@@ -81,15 +81,15 @@
...
 			</p>
 			<p>
 				<input type="radio" name="mode" value="webcron" class="radio"
 					   id="backgroundjobs_webcron" <?php if ($_['backgroundjobs_mode'] === "webcron") {
 			print_unescaped('checked="checked"');
 		} ?>>
 				<label for="backgroundjobs_webcron">Webcron</label><br/>
-				<em><?php p($l->t("cron.php is registered at a webcron service to call cron.php every 5 minutes over HTTP.")); ?></em>
+				<em><?php p($l->t("cron.php is registered at a webcron service to call cron.php every 5 minutes over HTTP. Usecase: Very small instance (1-5 users depending on the usage).")); ?></em>
```
Comment (Valdnet) [inline apps/settings/templates/settings/admin/server.php:92]:
```suggestion
				<em><?php p($l->t("cron.php is registered at a webcron service to call cron.php every 5 minutes over HTTP. Use case: Very small instance (1-5 users depending on the usage).")); ?></em>
```

**Comment by szaimen** [comment]
Thanks @Valdnet! done :)

File: apps/settings/templates/settings/admin/server.php [hunk 5]
```diff
@@ -81,15 +81,15 @@
...
+				<em><?php p($l->t("Execute one task with each page loaded. Use case: Single user instance.")); ?></em>
 			</p>
 			<p>
 				<input type="radio" name="mode" value="webcron" class="radio"
 					   id="backgroundjobs_webcron" <?php if ($_['backgroundjobs_mode'] === "webcron") {
 			print_unescaped('checked="checked"');
 		} ?>>
 				<label for="backgroundjobs_webcron">Webcron</label><br/>
-				<em><?php p($l->t("cron.php is registered at a webcron service to call cron.php every 5 minutes over HTTP.")); ?></em>
+				<em><?php p($l->t("cron.php is registered at a webcron service to call cron.php every 5 minutes over HTTP. Use case: Very small instance (1-5 users depending on the usage).")); ?></em>
```
Comment (Valdnet) [inline apps/settings/templates/settings/admin/server.php:92]:
```suggestion
				<em><?php p($l->t("cron.php is registered at a webcron service to call cron.php every 5 minutes over HTTP. Use case: Very small instance (1–5 users depending on the usage).")); ?></em>
```

**Review by Valdnet — COMMENTED** [review]
The only thing left to do is to change "-" into a half-pause.

File: apps/settings/templates/settings/admin/server.php (hunk 5 above)
Comment (szaimen) [reply apps/settings/templates/settings/admin/server.php:92]: great, thanks for this! I see you have typo knowledge :)

**Review by Valdnet — APPROVED** [review]

**Review by CarlSchwan — APPROVED** [review]

**Pull request merged**